bindings/python/iota_sdk/types/committee.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2024 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import List, Optional
+from dataclasses import dataclass, field
+from dataclasses_json import config
+from iota_sdk.types.common import EpochIndex, HexStr, json
+
+
+@json
+@dataclass
+class CommitteeMember:
+    """Information of a committee member.
+
+    Attributes:
+        address: Account address of the validator.
+        pool_stake: The total stake of the pool, including delegators.
+        validator_stake: The stake of a validator.
+        fixed_cost: The fixed cost of the validator, which it receives as part of its Mana rewards.
+    """
+    address: HexStr
+    pool_stake: int = field(metadata=config(
+        encoder=str
+    ))
+    validator_stake: int = field(metadata=config(
+        encoder=str
+    ))
+    fixed_cost: int = field(metadata=config(
+        encoder=str
+    ))
+
+
+@json
+@dataclass
+class Committee:
+    """The validator information of the committee.
+    Response of GET /api/core/v3/committee
+
+    Attributes:
+        committee: The validators of the committee.
+        total_stake: The total amount of delegated and staked IOTA coins in the selected committee.
+        total_validator_stake: The total amount of staked IOTA coins in the selected committee.
+        epoch: The epoch index of the committee.
+    """
+    committee: List[CommitteeMember]
+    total_stake: int = field(metadata=config(
+        encoder=str
+    ))
+    total_validator_stake: int = field(metadata=config(
+        encoder=str
+    ))
+    epoch: EpochIndex
+
+
+@json
+@dataclass
+class Validator:
```

Comment:
I would add a `Response` prefix to all responses like in Rust, this is too confusing otherwise, this is not a validator

Reply:
I agree